src/Spice86/Views/MemoryView.axaml

```diff
@@ -8,23 +8,34 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:rendering="clr-namespace:AvaloniaHex.Rendering;assembly=AvaloniaHex"
     xmlns:viewModels="clr-namespace:Spice86.ViewModels"
+    xmlns:userControls="clr-namespace:Spice86.UserControls"
     d:DesignHeight="450"
     d:DesignWidth="800"
     x:CompileBindings="True"
     x:DataType="viewModels:MemoryViewModel"
     mc:Ignorable="d">
     <ScrollViewer>
         <Grid RowDefinitions="Auto,*">
-            <Menu Grid.Row="0" IsVisible="{Binding IsPaused}">
-                <MenuItem Header="Edit">
-                    <MenuItem Command="{Binding EditMemoryCommand}" Header="Edit memory..." />
-                </MenuItem>
-            </Menu>
+            <WrapPanel Orientation="Horizontal" Grid.Row="0">
+                <Button Command="{Binding EditMemoryCommand}" Content="Edit..." />
+                <Button Command="{Binding DumpMemoryCommand}" Content="Dump..." />
+                <Button
+                        Command="{Binding NewMemoryViewCommand}"
+                        Content="New Memory View"
+                        IsEnabled="{Binding IsPaused}" />
+                <UniformGrid>
+                    <Label Content="Start" />
+                    <NumericUpDown Value="{Binding StartAddress}"/>
```

Comment:
For some reason these are ignoring manual editing of the address, except deletion.